### PR TITLE
Measure Tools: Remove the use of IModelApp feature tracking

### PR DESCRIPTION
**Pull request by calebmshafer**
With the impending removal in iTwin.js 5.0, https://github.com/iTwin/itwinjs-core/issues/5722, remove the iModelApp usage.

### CODE REVIEW

**Review by bsy-nicholasw — APPROVED** [review]

File: packages/itwin/measure-tools/src/api/FeatureTracking.ts
Comment (aruniverse) [inline packages/itwin/measure-tools/src/api/FeatureTracking.ts:1]: im not opposed to completely nuking this file, idk anyone who is actually using this.

@simnorm @a-gagnon @bsy-nicholasw @jjbeckman13 @jason-crow

**Review by aruniverse — APPROVED** [review]
missing rush change

**Review by ben-polinsky — APPROVED** [review]

File: packages/itwin/measure-tools/src/api/FeatureTracking.ts
Comment (bsy-nicholasw) [reply packages/itwin/measure-tools/src/api/FeatureTracking.ts:1]: It came from our discussion on internal API usage over at https://github.com/iTwin/itwinjs-backlog/issues/782#event-9866856242

My understanding after seeing these changes and pineapple's wiki on the matter is that the role is reversed, pineapple will explicitly hook into feature tracking events from the packages they consume?

Comment (calebmshafer) [reply packages/itwin/measure-tools/src/api/FeatureTracking.ts:1]: Yes, exactly. The pattern we want to move towards is that the apps would hook into events and do their feature tracking based on it.

However, I do think we should potentially remove all of the GUIDs and other parts that are specific.

**Comment by calebmshafer** [comment]
> missing rush change 

pushed as a minor change.

**Comment by aruniverse** [comment]
@wilmaier @AnDuong249 lets chat later about this

**Comment by pmconne** [comment]
@aruniverse @calebmshafer what's the status of this? It's mentioned as required by https://github.com/iTwin/itwinjs-backlog/issues/1103.

**Comment by calebmshafer** [comment]
> @aruniverse @calebmshafer what's the status of this? It's mentioned as required by https://github.com/iTwin/itwinjs-backlog/issues/1103.

@pmconne needed verification from @jason-crow if I recall correctly. Although haven't had a chance to get back to it for months.

**Comment by aruniverse** [comment]
> > @aruniverse @calebmshafer what's the status of this? It's mentioned as required by [iTwin/itwinjs-backlog#1103](https://github.com/iTwin/itwinjs-backlog/issues/1103).
> 
> @pmconne needed verification from @jason-crow if I recall correctly. Although haven't had a chance to get back to it for months.

Will merge this pr, and refactor the feature tracking in a separate issue.

**Pull request merged**